Source/WebCore/style/StyleScope.h

```diff
@@ -74,6 +74,10 @@ class Scope : public CanMakeWeakPtr<Scope> {
 
     ~Scope();
 
+    // For CheckedPtr / CheckedRef use.
+    void incrementPtrCount() const;
```

Comment:
We should just make this class inherit from CanMakeCheckedPtr instead unless this is absolutely required.

Reply:
I'll subclass CanMakeCheckedPtr. I missed the ShadowRoot ownership.